src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx

```diff
@@ -148,22 +148,25 @@ export default function ExecutionMetadata(): React.ReactElement {
 
   const renderSingleStageExecutionInfo = (): React.ReactElement | null => {
     const countLen = defaultTo(pipelineExecutionSummary?.stagesExecuted?.length, 0)
+    const shouldShowPopover = countLen - LIMIT > 0
```

Comment:
easy to read/understand
```suggestion
    const shouldShowPopover = countLen > LIMIT
```

Reply:
done